documentation/modules/post/windows/gather/forensics/fanny_bmp_check.md

```diff
@@ -0,0 +1,148 @@
+    [+] WORKSTATION1: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\ECELP4\filter3 found in registry.
+    [+] WORKSTATION1: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\ECELP4\filter8 found in registry.
+    [*] WORKSTATION1: 4 result(s) found in registry.
+
+-------------------------
+
+## Options
+List each option and how to use it.
+Session is needed, Do not know really what to include here, other than that.
+
+  < Intentionally left blank >
+
+-------------------------
+
+
+### Option Name
+
+Talk about what it does, and how to use it appropriately. If the default value is likely to change, include the default value here.
+
+< Intentionally left blank >
+
+-------------------------
+
+
+## Scenarios
+Specific #demo
+
+### Typical Usage Scenario: Pen Test without being Detected
+
+1 thing this could be used as, is (as with the duqu_check) to check,
+if a target system (that you/your team is going to/will/plan to perform one or
+more penetration tests on, already is infected by any of those, and it would
+ probably make a nice looking "alert" to malware researchers who runs malwares
+ into sandboxes and vms; (Because it would shortly make the system/vm
+ more "targeted" if infected even more.
+
+
+For example and if a VM(for example) is infected with Duqu,
+Maybe it's not the most optimal 'thing' to infect it with anything
+(else - in general, at all) By e.g Using metasploit,
+(because, the system already is infected with
+Duqu(if we take a Duqu Infected VM/System as an example),
+Duqu - wich is kinda well known by now, will make the VM
+(or if it is a real os, which still does happen) more suspected for malware.
+
+
+### So, short story: the less malicious activity (the less "malware") on a system, the less detection risk is present.
+
+I Will upload a POC video demonstrating this on Windows 10, x64. Sooner or later.
+There's already a XP POC video located here
+ - https://github.com/loneicewolf/fanny.bmp/blob/main/FannyMalware%20POC%20.mp4
+
+"Equation Group Q&A PDF File" - Explaining (not only) Fanny, (but also many others, in the "same family" of malware)
+
+ - https://github.com/loneicewolf/fanny.bmp/blob/main/Equation_group_questions_and_answers.pdf
+
+If needed, I included malware samples on the same page.
+
+ **It goes without saying that if you proceed to this page Please exercise caution.**
+
+ - https://github.com/loneicewolf/fanny.bmp
+
+
+## References
+
+- [x] https://securelist.com/a-fanny-equation-i-am-your-father-stuxnet/68787
+- [x] https://fmnagisa.wordpress.com/2020/08/27/revisiting-equationgroups-fanny-worm-or-dementiawheel/
+- [x] https://edwardsnowden.com/wp-content/uploads/2017/06/FOXACID-Server-SOP-Redacted.pdf
+
+
+### Optional reading
+
+- [x] https://github.com/loneicewolf/fanny.bmp/blob/main/2004.06195.pdf
+
+-------------------------
+
+
+## Version and OS
+
+### Tested on
+
+  - [x] Windows XP Pro SP3 English
```

Comment:
The documentation should contain a `Scenarios` section which contains an example run of this module on the specified platform.